examples/01_track/plot_01_overview_skore_project.py

```diff
@@ -336,8 +336,7 @@ def my_func(x):
 # Storing scikit-learn models and pipelines
 # =========================================
 #
-# As skore is developed by `Probabl <https://probabl.ai>`_, the spin-off of
-# scikit-learn, skore treats scikit-learn models and pipelines as first-class citizens.
+# Skore treats scikit-learn models and pipelines as first-class citizens.
```

Comment:
@tuscland in the related issue, you say "We just need to keep everything that's after the last comma.", but I think that the result is actually quite weird. Can we remove this sentence all together?

Reply:
Yes